### PR TITLE
Modify rule S5914: LaYC format

### PR DESCRIPTION
**Pull request by guillaume-dequenne-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [x] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by joke1196 — APPROVED** [review]
LGTM

**Review by maksim-grebeniuk-sonarsource — APPROVED** [review]
LGTM

File: rules/S5914/python/rule.adoc [hunk 1]
```diff
@@ -1,6 +1,8 @@
+This rule raises an issue when an assertion result is constant.
+
 == Why is this an issue?
 
-Assertions are meant to detect when code behaves as expected. An assertion which fails or succeeds all the time should be fixed.
+Assertions are meant to detect when code behaves as expected. An assertion which fails or succeeds all the time does not achieve this. Either it is redundant and should be removed to improve readabity or it is a mistake and the assertion should be corrected.
 
 
 This rule raises an issue when an assertion method is given parameters which will make it succeed or fail all the time. It covers three cases:
```
Comment (jean-jimbo-sonarsource) [inline rules/S5914/python/rule.adoc:8]: Amend the second case to "...called with a value which will always be None or never be None."

**Review by jean-jimbo-sonarsource — APPROVED** [review]
Left a suggestion on grammar

**Pull request merged**